packages/@lwc/wds-core/src/node/plugins/lwc/compile.js

```diff
@@ -89,15 +89,19 @@ export default ({ rootDir, moduleDirs }) => ({
   },
 
   resolveImport({ source, context, code, column, line }) {
-    const filePath = getRequestFilePath(context.url, rootDir);
+    const parentModuleFilePath = getRequestFilePath(context.url, rootDir);
```

Comment:
The variable name here was confusing. `filePath` is really the path of the _parent_ module - the module that is importing something. That wasn't clear from the variable name, and it confused me for a couple of minutes. Hopefully this code is clearer now.